src/CalcViewModel/HistoryViewModel.h

```diff
@@ -68,7 +68,7 @@ namespace CalculatorApp
             void UpdateHistoryVectorLength(_In_ int newValue, _In_ CalculationManager::CALCULATOR_MODE cMode);
             bool IsValid(_In_ CalculationManager::HISTORYITEM item);
 
-            void MakeHistoryClearedNarratorAnnouncement(Platform::String ^ resourceKey, Platform::String ^ &formatVariable);
+            Platform::String ^ MakeHistoryClearedNarratorAnnouncement(Platform::String ^ resourceKey, Platform::String ^ formatVariable);
```

Comment:
I don't think this change is necessary?

Reply:
Unfortunately, this change is necessary for the long-term.